ParsedownExtra.php

```diff
@@ -433,18 +453,27 @@ protected function parseAttributeData($attributeString)
     {
         $Data = array();
 
-        $attributes = preg_split('/[ ]+/', $attributeString, - 1, PREG_SPLIT_NO_EMPTY);
+        preg_match_all('/[^\s"]+(?:"[^"]*")?/', $attributeString, $attributes);
+
+        if( count($attributes) ) $attributes = $attributes[0];
+        else return $Data;
 
         foreach ($attributes as $attribute)
         {
             if ($attribute[0] === '#')
             {
                 $Data['id'] = substr($attribute, 1);
             }
-            else # "."
+            elseif ( $attribute[0] === '.' )
             {
                 $classes []= substr($attribute, 1);
             }
+            elseif ( strpos($attribute, '=') )
```

Comment:
Could you additionally add `and ! $this->safeMode` so that arbitrary attribute names aren't allowed if Parsedown-Extra is dealing with untrusted user input. Thanks! :)